cirq/google/common_serializers.py

```diff
@@ -336,3 +336,26 @@ def _near_mod_2(e, t, atol=1e-8):
         gate_constructor=lambda: ops.FSimGate(theta=-np.pi / 4, phi=0),
         args=[]),
 ]
+
+#
+# WaitGate serializer and deserializer
+#
+WAIT_GATE_SERIALIZER = op_serializer.GateOpSerializer(
+    gate_type=ops.WaitGate,
+    serialized_gate_id='wait',
+    args=[
+        op_serializer.SerializingArg(
+            serialized_name='nanos',
+            serialized_type=float,
+            gate_getter=lambda e: e.duration.total_nanos()),
```

Comment:
I wonder if we should put a limit here, like don't allow waits of more than 10 milliseconds (or should this be on the pyle side)?

Reply:
Yeah, that'd be a good idea.

Reply:
At least maybe yell if this is negative for now.